concent_api/utils/logging.py

```diff
@@ -16,272 +21,220 @@ def wrap(*args, **kwargs):
+        raise Exception
+    logger.info(f'File assigned to TASK_ID: {task_id} SUBTASK_ID: {subtask_id} is already uploaded. -- REQUESTOR PUBLIC KEY: {requestor_public_key} -- PROVIDER PUBLIC KEY {provider_public_key}')
 
 
-def log_request_received(path_to_file, operation: FileTransferToken.Operation):
-    logger.debug("{} request received. Path to file: '{}'".format(
-        operation.capitalize(),
-        path_to_file
-    ))
+def log_request_received(
+    logger,
+    path_to_file,
+    operation: FileTransferToken.Operation
+):
+    logger.debug(f"{operation.capitalize()} request received. Path to file: '{path_to_file}'")
 
 
 @replace_element_to_unavailable_instead_of_none
 def log_message_under_validation(
+    logger,
     operation: FileTransferToken.Operation,
     message_type: str,
     file_path: str,
     subtask_id: bytes,
     public_key: bytes
 ):
-
-    logger.debug("{} request will be validated. Message type: '{}'. File: '{}', with subtask_id '{}'. Client public key: '{}'".format(
-        operation.capitalize(),
-        message_type,
-        file_path,
-        subtask_id,
-        public_key
-    ))
+    logger.debug(f"{operation.capitalize()} request will be validated. Message type: '{message_type}'. File: '{file_path}', with subtask_id '{subtask_id}'. Client public key: '{public_key}'")
 
 
 @replace_element_to_unavailable_instead_of_none
 def log_message_successfully_validated(
+    logger,
     operation: FileTransferToken.Operation,
     message_type: str,
     file_path: str,
     subtask_id: bytes,
     public_key: bytes
 ):
-    logger.info("{} request passed all validations.  Message type: '{}'. File: '{}', with subtask_id '{}'. Client public key: '{}'".format(
-        operation.capitalize(),
-        message_type,
-        file_path,
-        subtask_id,
-        public_key
-    ))
+    logger.info(f"{operation.capitalize()} request passed all validations.  Message type: '{message_type}'. File: '{file_path}', with subtask_id '{subtask_id}'. Client public key: '{public_key}'")
 
 
 @replace_element_to_unavailable_instead_of_none
 def log_operation_validation_failed(
+    logger,
     operation: FileTransferToken.Operation,
     message: str,
     error_code: str,
     path: str,
     subtask_id: str,
     client_key: str
 ):
-    logger.info("{} validation failed. Message: {} Error code: '{}'. File '{}', with subtask_id '{}'. Client public key: '{}'".format(
-        operation.capitalize(),
-        message,
-        error_code,
-        path,
-        subtask_id,
-        client_key
-    ))
+    logger.info(f"{operation.capitalize()} validation failed. Message: {message} Error code: '{error_code}'. File '{path}', with subtask_id '{subtask_id}'. Client public key: '{client_key}'")
+
+
+def _get_field_value_from_messages_for_logging(
+    field_name: MessageFields,
+    message: Message
+)->str:
+    return get_field_from_message(message, field_name.value) if message is not None else 'UNAVAILABLE'
```

Comment:
How do you discern `UNAVAILABLE` from a valid ID? This could be a valid ID in a message. If someone decides to use it for any reason, it will be very tricky to find out what's happening. Get ready for a night of confused debugging :)

You should do it in such a way that the ID can never look like this special value. For example always put something around around a normal ID (`'1'`, `'some id'`, `'UNAVAILABLE'`) and never around this special value (`UNAVAILABLE`).

Reply:
done and changed in all functions in file.